python/plugin.xml

```diff
@@ -6,40 +6,17 @@
       <language
             class="org.libreoffice.ide.eclipse.python.Language"
             name="Python">
-         <newWizardPage
-               class="org.libreoffice.ide.eclipse.python.PythonWizardPage">
-         </newWizardPage>
       </language>
    </extension>
    <extension
          id="org.libreoffice.ide.eclipse.python.builder"
          point="org.eclipse.core.resources.builders">
       <builder/>
    </extension>
-   <extension
-         point="org.libreoffice.ide.eclipse.core.MainProvider">
-      <MainProvider class="org.libreoffice.ide.eclipse.python.PythonMainProvider"/>
-   </extension>
+   
    <extension
          point="org.eclipse.ui.startup">
       <startup class="org.libreoffice.ide.eclipse.python.PythonUnoResourceChangeHandler"/>
    </extension>
-   <extension
-         point="org.eclipse.jdt.core.classpathContainerInitializer">
-      <classpathContainerInitializer
-            class="org.libreoffice.ide.eclipse.python.build.OooClasspathContainerInitializer"
-            id="org.libreoffice.ide.eclipse.java.OOO_CONTAINER">
-      </classpathContainerInitializer>
-   </extension>
-   <extension
-         point="org.eclipse.jdt.ui.classpathContainerPage">
-      <classpathContainerPage
-            class="org.libreoffice.ide.eclipse.python.build.OOoContainerPage"
-            id="org.libreoffice.ide.eclipse.java.OOO_CONTAINER"
-            name="LibreOffice Libraries">
-      </classpathContainerPage>
-   </extension>
-   
-   
 
```

Comment:
The 
org.libreoffice.ide.eclipse.python.build
    |-class OOoContainerPage
    |-class OooClasspathContainerInitializer
    |-class OOoClasspathContainer
    |-class Messages
    |-messages.properties
classes and the properties file mentioned under org.libreoffice.ide.eclipse.python.build are required if there are LibreOffice Libraries under the build path. In that case you might need to edit the libraries which are not required in the Python extension project. Therefore Removing them from the Python module. And since we are removing the classes so their respective entries are being removed plugin.xml.